main.go

```diff
@@ -91,6 +125,53 @@ func cleanupImages(ecrCli *ecr.ECR, repoName string, images []*ecr.ImageDetail,
 	return nil
 }
 
+func excludeImages(srcImages []*ecr.ImageDetail, excludeImages []*ecr.ImageDetail) (result []*ecr.ImageDetail) {
+	var addImg bool
+	result = srcImages[:0]
```

Comment:
Is  this necessary? result will initialize to nil and append will create the slice.

Note: You could reduce the asymptotic complexity of this algo by using maps, but since we know there'll only be 1k images at most. this is fine.